apps/extension/src/ui/domains/Settings/MigratePassword/Error.tsx

```diff
@@ -25,9 +25,9 @@ export const MigratePasswordError = () => {
                 team on{" "}
                 <a
                   className="text-white opacity-100"
-                  href="https://discord.gg/2EmmfrTN"
+                  href="https://discord.gg/talisman"
```

Comment:
The old URL doesn't work anymore, as far as I can tell

Reply:
![image](https://github.com/TalismanSociety/talisman/assets/2741569/f829dec2-54ce-4b65-a537-8b8069819a61)